README.md

```diff
@@ -182,6 +182,18 @@ SELECT u.id FROM users as u
 
 will return `u.id` instead of just `id` if `columnsWithAlias=true`.
 
+##### `interpolateParams`
+
+```
+Type:           bool
+Valid Values:   true, false
+Default:        false
+```
+
+When `interpolateParams` is true, calls to `sql.Db.Query()` and `sql.Db.Exec()` with params interpolates placeholders (`?`) with given params. This reduces roundtrips to database compared with `interpolateParams=false` since it uses prepare, exec and close to support parameters.
+
+NOTE: It make SQL injection vulnerability when connection encoding is multibyte encoding except utf-8 (e.g. cp932).
```

Comment:
Hi guys, I really would like to see a test case for this as it's not 100% obvious for me how this vulnerability applies to this go mysql driver.
Here is a more about it: http://stackoverflow.com/a/12118602/916440
I've also just noticed that usage of `charset` is discouraged in favor of `collation` https://github.com/go-sql-driver/mysql#charset - a little surprise as for me `charset` and `collation` were always two different things http://dev.mysql.com/doc/refman/5.6/en/charset-general.html - but not an expert on that matter. Just pointing out I would love to see testcase for this sql injection issue plus explanation on charset vs collation.